packages/patternlab/styleguide/source/assets/js/modules/utilNav.js

```diff
@@ -1,9 +1,34 @@
-export default (function (window, document, $, undefined) {
+import focusTrapping from "../helpers/focusTrapping.js";
 
+export default (function (document,$) {
+  const wideContainerClass = '.js-utility-nav--wide .js-util-nav-content';
+  const narrowContainerClass = '.js-utility-nav--narrow .js-util-nav-content';
   let $panels = $('.js-util-nav-content');
   let $panel = "";
   let $utilityButtons = $('.js-util-nav-toggle');
 
+  // Keyboard navigation.
+  $(document).keydown(function(e) {
+    // check if menu open
+    if (!$(wideContainerClass).hasClass('is-closed')) {
+      focusTrapping({
+        focusableSelectors: '.js-utility-nav--wide .js-util-nav-content a, .js-utility-nav--wide .js-util-nav-content button',
+        modalSelector: '.js-utility-nav--wide .js-util-nav-content',
+        keyEvent: e
+      });
+    }
+
+    // check if menu open
+    if (!$(narrowContainerClass).hasClass('is-closed')) {
+      focusTrapping({
+        focusableSelectors: '.js-utility-nav--narrow .js-util-nav-content a, .js-utility-nav--narrow .js-util-nav-content button',
+        modalSelector: '.js-utility-nav--narrow .js-util-nav-content',

```

Comment:
```suggestion
        modalSelector: '.js-utility-nav--narrow .js-util-nav-content',
```suggestion
        modalSelector: narrowContainerClass,

```
```

Reply:
actually, I don't think you want to do this with the narrow container on mobile. This should be handled together in the main nav in the mobile tray, otherwise, the focus is only trapped inside of the utility nav items. Can you remove this function altogether instead?